packages/plugins/io-center/src/command.svelte.ts

```diff
@@ -54,16 +126,25 @@ export class Command {
 		}
 		const ld0 = this.ensureLD0(ied)
 
-		const ln = store.doc.createElement('LN')
-		ln.setAttribute('lnType', type)
-		ln.setAttribute('lnClass', type)
-		ln.setAttribute('inst', instance)
+		const attributes = {
+			"lnClass": type,
+			"lnType": type,
+			"inst": instance,
+		}
 
-		const editEvent = { node: ln, parent: ld0, reference: null }
-		createAndDispatchEditEvent({
-			host,
-			edit: editEvent,
-		})
+		createElement(host, store.doc, "LN", attributes, ld0, null)
+	}
+
+	private ensureSelectedIED(): Element {
```

Comment:
nit: ensure usually means that if the thing does not exist yet, you create it.
I've used "require" like `requireHost` to throw and error if it is not there.